### PR TITLE
Updated buttons on Unsaved Modification window

### PR DESCRIPTION
**Pull request by rsredsq**
#438

### CODE REVIEW

File: Resources/EditorData/AtomicEditor/resources/language/lng_en.tb.txt [hunk 1]
```diff
@@ -14,3 +14,4 @@ new New
 save Save
 close Close
 search Search
+dont_save Don't save
```
Comment (JoshEngebretson) [inline Resources/EditorData/AtomicEditor/resources/language/lng_en.tb.txt]: Don't Save (capitalization)

File: Source/AtomicEditor/Editors/ResourceEditor.cpp [hunk 2]
```diff
@@ -42,11 +42,15 @@ class EditorTabLayout: public TBLayout
         if (editor_->HasUnsavedModifications())
         {
             TBMessageWindow *msg_win = new TBMessageWindow(this, TBIDC("unsaved_modifications_dialog"));
-            TBMessageWindowSettings settings(TB_MSG_OK_CANCEL, TBID(uint32(0)));
+            TBMessageWindowSettings settings(TB_MSG_NONE, TBID(uint32(0)));
             settings.dimmer = true;
             settings.styling = true;
             String windowString = Atomic::ToString("%s has unsaved modifications.\nDo you wish to discard them and close?", GetFileNameAndExtension(editor_->GetFullPath()).CString());
             msg_win->Show("Unsaved Modifications",  windowString.CString(), &settings, 640, 360);
+            TBButton *btn = new TBButton();
```
Comment (JoshEngebretson) [inline Source/AtomicEditor/Editors/ResourceEditor.cpp]: Where is btn referenced?

File: Source/AtomicEditor/Editors/ResourceEditor.cpp [hunk 3]
```diff
@@ -42,11 +42,15 @@ class EditorTabLayout: public TBLayout
...
+            msg_win->AddButton("dont_save", false, msg_win->GetWidgetByIDAndType<TBLayout>(5));
```
Comment (JoshEngebretson) [inline Source/AtomicEditor/Editors/ResourceEditor.cpp]: GetWidgetByIDAndType<TBLayout>(5)

Kind of magical, we're now using internal id's in external code, and there are enough of them that this gets confusing

**Comment by JoshEngebretson** [comment]
Ok, this looks good... let's fix the capitalization on "Don't Save" and remove:

    TBButton *btn = new TBButton();

If it is not needed...

File: Resources/EditorData/AtomicEditor/resources/language/lng_en.tb.txt (hunk 1 above)
Comment (rsredsq) [inline Resources/EditorData/AtomicEditor/resources/language/lng_en.tb.txt]: ok

File: Source/AtomicEditor/Editors/ResourceEditor.cpp (hunk 2 above)
Comment (rsredsq) [inline Source/AtomicEditor/Editors/ResourceEditor.cpp]: sorry, no need that

**Comment by JoshEngebretson** [comment]
Ok, looks good.  Landing!

**Pull request merged**